src/aind_metadata_service/sharepoint/nsb2023/mapping.py

```diff
@@ -66,6 +66,7 @@ class HeadPost(Enum):
     WHC_NP_ZIRCONIA = "WHC NP (Zirconia)"
     L_SHAPED = "L-shaped"
     LC = "LC"
+    DHC = "DHC"
```

Comment:
I'm confused by the two enums for HeadPost and HeadPostType. What's the difference? How are they being used?

Reply:
In the request form they have 2 entry fields for "headpost" and "headpost type". We're using both of them in method `from_hp_and_hp_type" to infer information about the headframe and well.